modules/editorial-metadata/lib/components/editorial-metadata-manager.js

```diff
@@ -0,0 +1,181 @@
+			);
+		} catch ( error ) {
+			handleErrorThrown( error.message );
+		}
+	};
+
+	const deleteModal = (
+		<ConfirmDeleteModal
+			confirmationMessage={ '' }
+			dataType={ 'metadata field' }
+			name={ eMetadataTerm?.name }
+			onCancel={ () => setIsConfirmingDelete( false ) }
+			onConfirmDelete={ handleDelete }
+		/>
+	);
+
+	const createEditModal = (
+		<CreateEditEditorialMetadataModal
+			availableMetadataTypes={ availableMetadataTypes.map( availableMetadataType => {
+				return {
+					value: availableMetadataType,
+					label: availableMetadataType,
+				};
+			} ) }
+			metadata={ eMetadataTerm }
+			onCancel={ () => setIsModalVisible( false ) }
+			onSuccess={ handleSuccess }
+			onErrorThrown={ handleErrorThrown }
+		/>
+	);
+
+	return (
+		<>
+			{ <SuccessNotice success={ success } /> }
+			{ error && (
+				<div style={ { marginBottom: '1rem' } }>
+					<Notice status="error" isDismissible={ true } onRemove={ () => setError( null ) }>
+						<p>{ error }</p>
+					</Notice>
+				</div>
+			) }
+			<Flex direction={ [ 'column' ] } justify={ 'start' } align={ 'start' }>
+				<FlexItem>
+					<Button
+						variant="secondary"
+						onClick={ () => {
+							setEMetadataTerm( null );
+							setIsModalVisible( true );
+						} }
+					>
+						{ __( 'Add New Metadata', 'vip-workflow' ) }
+					</Button>
+				</FlexItem>
+				<Flex className="emetadata-items" direction={ [ 'column', 'row' ] } justify={ 'start' }>
+					{ eMetadataTerms.map( eMetadataTerm => {
+						return (
+							<FlexItem className="emetadata-item" key={ eMetadataTerm.term_id }>
+								<Card>
+									<CardHeader>
+										<Flex direction={ [ 'column' ] } justify={ 'start' } align={ 'start' }>
+											<Heading level={ 4 }>{ eMetadataTerm.name }</Heading>
+											<Text>
+												<i>{ eMetadataTerm.description }</i>
+											</Text>
+										</Flex>
+										<Flex direction={ [ 'column', 'row' ] } justify={ 'end' } align={ 'end' }>
+											<div className="crud-emetadata">
+												<Button
+													className="delete-emetadata"
+													variant="secondary"
+													onClick={ () => {
+														setEMetadataTerm( eMetadataTerm );
+														setIsConfirmingDelete( true );
+													} }
+													style={ {
+														color: '#b32d2e',
+														boxShadow: 'inset 0 0 0 1px #b32d2e',
+													} }
+												>
+													{ __( 'Delete', 'vip-workflow' ) }
+												</Button>
+												<Button
+													variant="primary"
+													onClick={ () => {
+														setEMetadataTerm( eMetadataTerm );
+														setIsModalVisible( true );
+													} }
+												>
+													{ __( 'Edit', 'vip-workflow' ) }
+												</Button>
+											</div>
+										</Flex>
+									</CardHeader>
+								</Card>
+							</FlexItem>
+						);
+					} ) }
+				</Flex>
+			</Flex>
+
+			{ isConfirmingDelete && deleteModal }
+			{ isModalVisible && createEditModal }
```

Comment:
`isModalVisible` is unclear as there are a couple of modals that this component uses. How about:

```suggestion
			{ isCreateEditModalVisible && createEditModal }
```

or 

```suggestion
			{ isEditingEditorialMetadata && createEditModal }
```

Reply:
Good point, I'll make that change.